templates/openwisp2/settings.py

```diff
@@ -142,6 +151,12 @@
 OPENWISP_RADIUS_SMS_TOKEN_MAX_IP_DAILY = {{ openwisp2_radius_sms_token_max_ip_daily }}
 SENDSMS_BACKEND = '{{ openwisp2_radius_sms_backend }}'
 
+# django-sesame configuration for magic sign-in links.
+# Refer https://github.com/aaugustin/django-sesame#django-sesame.
+AUTHENTICATION_BACKENDS += [
+    'sesame.backends.ModelBackend',
+]
+SESAME_MAX_AGE = timedelta(minutes=30)
```

Comment:
what's the point of having `openwisp2_django_sesame_max_age` if then 30 minutes is hardcoded here?